### PR TITLE
Hook calculat.js file to my application

### PR DESCRIPTION
**Pull request by addisalemseifu**
-Added the calculate.js file to my project.
-Added the operate.js file to my project.
-Installed big.js library.
-Hooked the calculate.js file to my APP.js file.
-Updated my README.Md file.
cc

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
+
+  const addTotext = (val) => {
+    setObj(calculate(obj, val));
+  };
+
   return (
     <div className="calc-grid">
-      <Output data-testid="my-output" />
+      <Output data-testid="my-output" total={obj.total} next={obj.next} />
       <Buttons type="button" symbol="AC" className="clear" />
```
Comment (joskalenda) [inline src/components/Calculator.js]: - [ ] When clinking on this button `AC`, the app returns this error below



- [ ] I think this button should reset the input field, please take a look at calculate.js and fix this issue

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @addisalemseifu , |
|Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there! |
|  |
 
## Highlights

- [ ].. ✅ Great design
- [ ].. ✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Buttons.js [hunk 2]
```diff
@@ -3,10 +3,11 @@ import '../index.css';
 
 export default function Buttons({
   // eslint-disable-next-line react/prop-types
```
Comment (Gambit142) [inline src/components/Buttons.js:5]: - [OPTIONAL] I would advise you to remove the `eslint-disable` statement. `Eslint` helps us to write industry-standard, clean and readable codes. Using `eslint-disable` statement defeats this purpose, kindly remove it. 👍 💻

**Review by Gambit142 — APPROVED** [review]
Hi @addisalemseifu 👋,

## Status: Approved 🟢   

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

### Highlights
- All linters checks passed ✔️ 
- Great use of the logic files for calculator functionality ✔️ 
- Application works properly ✔️ 

### Optional Suggestion

- #### Kindly consider the comment tagged [OPTIONAL] under this review summary 😎 

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**